.changeset/grumpy-fishes-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+"kubernetes-agent": patch
```

Comment:
I think this can be a minor bump. It's new functionality

Reply:
Done